README.md

```diff
@@ -94,73 +138,103 @@ firmwares/
+  * this will be "yes" when running on the buildbot farm and helps to save some
+  disc-space by removing files not required anymore. On manual builds you 
+  should not set this to "yes", as you have to rebuild the whole toolchain each
+  time.
 * SET_BUILDBOT :
   * "env" the Makefile will honor the "IS_BUILDBOT" environment
   * "yes" the Makefile will always act as "IS_BUILDBOT" was set to "yes"
-  * "no"  the Makefile will always act as "IS_BUILDBOT" was set to "no" / is unset. This way we can run builds on the buildbot like a local build.
+  * "no"  the Makefile will always act as "IS_BUILDBOT" was set to "no" / is unset. 
+    This way we can run builds on the buildbot like a local build.
+
+
+### Build a new router model for an existing architecture
+
+Everything that is in the [Table of Hardware](https://openwrt.org/toh/start)
+should work at least in theory. Wifi should be ath9k / ath10k. mt76 also appears
+to work. 
+
+You can use the Imagebuilder to get a list of all supported models for a
+platform. You can also look in the [source](https://git.openwrt.org/?p=openwrt/openwrt.git;a=tree;f=target/linux/ar71xx/image)
+for model names. 
+
+Just put the name in the `/profiles/<target>.profiles` and it should work. 
+
 
 ### Continuous integration / Buildbot
 
-The firmware is [built
-automatically](http://buildbot.berlin.freifunk.net/one_line_per_build) by our [buildbot farm](http://buildbot.berlin.freifunk.net/buildslaves). If you have a bit of CPU+RAM+storage capacity on one of your servers, you can provide a buildbot slave (see [berlin-buildbot](https://github.com/freifunk/berlin-buildbot)).
+The firmware is [built automatically](http://buildbot.berlin.freifunk.net/one_line_per_build)
+by our [buildbot farm](http://buildbot.berlin.freifunk.net/buildslaves). If you
+have a bit of CPU+RAM+storage capacity on one of your servers, you can provide
+a buildbot slave (see [berlin-buildbot](https://github.com/freifunk/berlin-buildbot)).
+
+All branches whose name complies to the "X.Y.Z" pattern are built and put into
+the "stable" downloads directory: [http://buildbot.berlin.freifunk.net/buildbot/stable/](http://buildbot.berlin.freifunk.net/buildbot/stable/)
 
-All branches whose name complies to the "X.Y.Z" pattern are built and put into the "stable" downloads directory:
-[http://buildbot.berlin.freifunk.net/buildbot/stable/](http://buildbot.berlin.freifunk.net/buildbot/stable/)
+All branches with names not fitting the "X.Y.Z" pattern are built and put into
+the "unstable" directory: [http://buildbot.berlin.freifunk.net/buildbot/unstable/](http://buildbot.berlin.freifunk.net/buildbot/unstable/)
+Note that in the directory there is no reference to the branch name; unstable
+builds can be identified by build number only.
```

Comment:
referencing the VERSION.txt file? As this is the best place to look up on which branch and revision a build is based on.